example/tsconfig.json

```diff
@@ -16,7 +16,7 @@
     "isolatedModules": true,
     "outDir": "tmp",
     "noEmit": true,
-    "types": ["node", "vite/client"],
+    "types": ["node", "vite/client", "@qwilkdev/pwa/head"],
```

Comment:
I don't like so much configuration, that's why we had @qwikdev/pwa/icons-entry which has its own types and it gets handled by the load hook in vite.

Reply:
maybe we can create a new composable like `useDocumentHead` to handle it internally